pkgs/by-name/ch/chirpstack-udp-forwarder/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, protobuf
+}: rustPlatform.buildRustPackage rec {
+  pname = "chirpstack-udp-forwarder";
+  version = "4.1.5";
+
+  src = fetchFromGitHub {
+    owner = "chirpstack";
+    repo = "chirpstack-udp-forwarder";
+    rev = "81dbf333d7d0fca08e025a2ed801a7592d1cecb8";
+    sha256 = "sha256-JRwWD5TJCibTNRmn69lBPT+aJ/91y92Y7+hbJKxc3BE=";
```

Comment:
use `hash` instead

Reply:
Thanks, I am still a Nixpkgs newbie :D I addressed it in my latest push.